shared/fs/common/path-item-action/choose-view.js

```diff
@@ -11,6 +11,7 @@ import Confirm from './confirm-container'
 type OwnProps = {|
   floatingMenuProps: FloatingMenuProps,
   path: Types.Path,
+  position?: 'row' | 'header',
```

Comment:
Just realized we might have done the similar thing: https://github.com/keybase/client/pull/17120/files#diff-b8167efbb256878db6c5717a2b0c1a94R15 I might change 'header' to 'screen' when rebasing mine, since this used in the file preview screen too (i.e. more than header)